tools/cli/go-whisk-cli/commands/qualifiedname.go

```diff
@@ -0,0 +1,207 @@
+/*
```

Comment:
I think the standard naming conventions for go files are to separate words using an underscore. Ie, quailifed_name.go.